docs/getting-started-guides/libvirt-coreos.md

```diff
@@ -46,8 +46,8 @@ On the other hand, `libvirt-coreos` might be useful for people investigating low
 4. Install [libvirt](http://libvirt.org/)
 5. Install [openssl](http://openssl.org/)
 6. Enable and start the libvirt daemon, e.g.:
-   * ``systemctl enable libvirtd && systemctl start libvirtd`` # for systemd-based systems
-   * ``/etc/init.d/libvirt-bin start`` # for init.d-based systems
+   * ``systemctl enable libvirtd && systemctl start libvirtd``  for systemd-based systems
+   * ``/etc/init.d/libvirt-bin start``  for init.d-based systems
```

Comment:
These two lines show up as numbered list (1, 2) and not bullets in the preview.

Reply:
It's normal, origin web show up as bullets in the preview also, but will show up as numbered list (1, 2) in webpage.

Reply:
would prefer if these were bullet points---can you try putting a new line before these bullet points? sometimes that fixes the markdown

Reply:
@abiogenesis-now Hmm. I tried to get this inner list to be bulleted. See the two most recent commits.But it still renders as numbered.